src/main/java/com/walking/intensive/chapter2/task6/Task6.java

```diff
@@ -19,8 +19,19 @@ public static void main(String[] args) {
      * <p>Если входные данные некорректны - метод должен возвращать -1.
      */
     static int getLcm(int m, int n) {
-        // Ваш код
-        return 0;
+        if (isInvalidValue(m, n)) {
+            return -1;
+        }
+
+        int minValue = Math.min(m, n);
+        int maxValue = Math.max(m, n);
+        for (int i = 1; i < maxValue ; i++) {
```

Comment:
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter2/task6/Task6.java:28:38: ';' is preceded with whitespace. [NoWhitespaceBefore]